plugins/modules/vmware_resource_pool.py

```diff
@@ -294,19 +315,25 @@ def main():
                               resource_pool=dict(required=True, type='str'),
                               mem_shares=dict(type='str', default="normal", choices=[
                                               'high', 'custom', 'normal', 'low']),
+                              mem_allocation_shares=dict(type='int'),
```

Comment:
Can we set some default value just like other parameters?

Reply:
I added to the default value based on the normal value on vCenter 7.